src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final FigureSupplier supplier = new FigureSupplier();
+
+    public static void main(String[] args) {
+        Figure[] figures = {
+                supplier.getRandomFigure(),
+                supplier.getRandomFigure(),
+                supplier.getRandomFigure(),
+                supplier.getDefaultFigure(),
+                supplier.getDefaultFigure(),
+                supplier.getDefaultFigure()
+        };

```

Comment:
use loop for creating figures 
https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#you-should-create-several-random-figures-so-you-will-use-a-loop-please-dont-create-a-new-figuresupplier-inside-the-loop